main.js

```diff
@@ -196,7 +195,7 @@ app.on('ready', function() {
 	Menu.setApplicationMenu(appMenu);
 
 	// We observe changes on the `rulers` array to enable/disable the toggle menu.
-	Array.observe(rulers, () => toggleRulersMenu.enabled = !!rulers.length);
+	Array.observe(rulers, () => toggleRulersMenu.enabled = Boolean(rulers.length));
```

Comment:
This syntax won't change the behavior, but makes the line more readable and the casting safer.